positron/components/PositronAppRunner.jsm

```diff
@@ -53,14 +54,14 @@ PositronAppRunner.prototype = {
     return this._unboundLoadAndContinue.bind(this, aThenWhat.bind(this));
   },
 
-  _loadMainScript: function par_LoadMainScript(aScriptName) {
-    let mainScript = this._baseDir.clone();
-    mainScript.append(aScriptName);
+  _loadMainScript: function par_LoadMainScript(aScriptPath) {
+    let mainScriptPath = OS.Path.join(this._baseDir.path, aScriptPath);
+    let mainScriptURI = OS.Path.toFileURI(mainScriptPath);
     let requirer = {
       id: "resource:///modules/PositronAppRunner.jsm",
       exports: {},
     };
-    (new Require(requirer))(Services.io.newFileURI(mainScript).spec);
+    (new Require(requirer))(mainScriptURI);
```

Comment:
Not something to fix here... but Require() is an object that instantiates to a function? That seems very odd O_o. Is it too late to fix this (i.e., should be `const require = makeRequire(requirer)` and not a constructor, or am I missing something)?